docs-source/content/userguide/managing-fmw-domains/soa-suite/Logging-Exporter-Setup.md

```diff
@@ -0,0 +1,126 @@
+# Weblogic Logging Exporter Setup to publish logs into Elasticsearch
+
+The WebLogic Logging Exporter adds a log event handler to WebLogic Server. 
+
+WebLogic Server logs can be pushed into Elastic Stack in Kubernetes directly, by using the Elasticsearch REST API. For more details refer to [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter) project.  
+
+This documents provides the steps to publish WebLogic Server logs into Elasticsearch and interact with them in Kibana. 
+
+Please follow the below steps to push the server server logs into Elasticsearch.  
+
+## Download Weblogic logging exporter binaries
+
+The Weblogic logging exporter pre-built binaries are available in the release page of the [weblogic-logging-exporter](https://github.com/oracle/weblogic-logging-exporter/releases) project.  
+You need to download the 
+* "weblogic-logging-exporter-0.1.1.jar" from this [link](https://github.com/oracle/weblogic-logging-exporter/releases)
+* "snakeyaml-1.23.jar" from [Maven Central](https://search.maven.org/artifact/org.yaml/snakeyaml/1.23/bundle).  
+
+## Copy JAR files to the Kubernetes Weblogic Domain Pod
+
+Copy the weblogic-logging-exporter-0.1.1.jar and snakeyaml-1.23.jar files to the domain home folder in the Admin server pod.
+
+```
+$ kubectl cp snakeyaml-1.23.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+ 
+$ kubectl cp weblogic-logging-exporter-0.1.1.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+```
+## Add a startup class to your domain configuration
+
+In the Administration Console, navigate to "Environment" then "Startup and Shutdown classes" in the main menu.  
+Add a new Startup class. You may choose any descriptive name and the class name must be "weblogic.logging.exporter.Startup".  
+
+![Add startup class](images/Logging-Exporter_add-startup-class.png)  
+
+Target the startup class to each server that you want to export logs from.  
+
+![Target the startup class](images/Logging-Exporter_target-startup-class.png)  
+
+You can verify this by checking for the update in your config.xml which should be similar to this example:
+```
+<startup-class>
+  <name>weblogic-logging-exporter</name>
+  <target>AdminServer,soa_cluster</target>
+  <class-name>weblogic.logging.exporter.Startup</class-name>
+</startup-class>
+```  
+
+## Update Weblogic Server CLASS Path.
+
+Copy setDomainEnv.sh from the pod to local folder.  
+```
+$ kubectl cp soans-au01/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/bin/setDomainEnv.sh .
```

Comment:
did you test this? this likely wont work

Reply:
What is `soans-au01`?  Is this the namespace?  If so, call it out and show a templatized version and then the sample.